libs/blocks/section-metadata/section-metadata.css

```diff
@@ -316,11 +319,13 @@ main > .section[class*='-up'] > .content {
   }
 }
 
-@media screen and (min-width: 1200px) {
+@media screen and (min-width: 1070px) {
```

Comment:
I'm not confident in this change. This is the only such media query in our code, so there must be something else at play here. A bit more investigation might be needed here - looking at the code, it seems that the `.#-up` pattern was only supposed to work on desktop, so it could be that the request from the ticket is not a bug, it's an enhancement that should come as a new requirement and get Consonant approval. Can you look through older PRs to check when the `.#-up` pattern was introduced and which request it came from?